model/ftn/ww3_grid.ftn

```diff
@@ -4906,21 +4910,29 @@
               NPO = NML_OUTBND_LINE(I)%NP
               I=I+1
             ELSE
-              EXIT
+!              EXIT
+! Bug (minor): If we exit here, a code block below will not be reached that
+! writes info about the last boundary file:'File nestI.ww3  Number of points ..'
+! ( The block does WRITE(NDSO,991), WRITE (NDSO,992), etc.)
+! Instead, let NPO=0 mark that we have got the info for the last file, and the
+! do-loop over outbound lines will exit immediately after the block. This is a
+! minor bug because it only affects output messages (C. Hansen 20190701)
+               NPO = 0
             END IF
           ELSE
             CALL NEXTLN ( COMSTR , NDSI2 , NDSE )
             READ (NDSI2,*,END=2001,ERR=2002) XO0, YO0, DXO, DYO, NPO
           END IF
 !
-          IF ( ILOOP .EQ. 1 ) THEN
-            IF (FLGNML) THEN
-              WRITE(NDSS,'(A)') NML_OUTBND_LINE(I-1)
-            ELSE
+          IF ( .NOT. FLGNML .AND. ILOOP .EQ. 1 ) THEN
+!            IF (FLGNML) THEN
+! NML_OUTBND_LINE are not strings (C. Hansen 20190701), but NDSS isn't used
+!              WRITE(NDSS,'(A)') NML_OUTBND_LINE(I-1)
+!            ELSE
```

Comment:
to remove and mention in the commit message

Reply:
I understand, thank you. I will do as you ask.